modules/installation-osp-provider-networks.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+
+[id="installation-osp-provider-networks_{context}"]
+= Cluster deployment on {rh-openstack} provider networks
+
+You can deploy your {product-title} clusters on {rh-openstack-first} with a primary network interface on a provider network. Provider networks are commonly used to give projects direct access to a public network that can be used to reach the Internet. You can also share provider networks among projects as part of the network creation process.
+
+{rh-openstack} provider networks map directly to an existing physical network in the data center. A {rh-openstack} administrator must create them.
+
+In the following example, {product-title} workloads are connected to a data center by using a provider network:
+
+image::openshift-on-openstack-provider-network.png[A diagram that depicts four OpenShift workloads on OpenStack. Each workload is connected by its NIC to an external data center by using a provider network.]
+
+{product-title} clusters that are installed on provider networks do not require tenant networks or floating IP addresses. The installer does not create these resources during installation.
+
+Example provider network types include flat (untagged) and VLAN (802.1Q tagged).
+
+[NOTE]
+====
+A cluster can support as many provider network connections as the network type allows. For example, VLAN networks typically support up to 4096 connections.
+====
+
+You can learn more about provider and tenant networks in link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/sec-networking-concepts#tenant-provider-networks[the {rh-openstack} documentation].
```

Comment:
Wasn't sure if this target doc just isn't live yet (it's 404'ing currently) or maybe the URL changed? Should it maybe be going to this?

https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/networking-concepts_networking-concepts#provider-networks_networking-concepts

Reply:
Might've moved. Thanks!

Reply:
```suggestion
You can learn more about provider and tenant networks in link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/networking-concepts_networking-concepts#provider-networks_networking-concepts[the {rh-openstack} documentation].
```